Makefile

```diff
@@ -102,19 +111,9 @@ release:
 	mv _oasis.tmp _oasis
 	oasis setup
 
-# make coverage - prepares for building with coverage analysis
-# make uncover  - reverses the setup from "make coverage"
 # make report   - create coverage/index.html 
 
-coverage: _tags _tags.coverage 
-	test ! -f _tags.orig && mv _tags _tags.orig || true
-	cat _tags.coverage _tags.orig > _tags
-
-uncover: _tags.orig
-	mv _tags.orig _tags
-
 report:
 	bisect-ppx-report -I _build -html coverage /tmp/bisect-xenops*out
```

Comment:
One issue not related to this PR is that this might use out-of-date coverage data that is generated for an earlier build. And the old coverage data is not cleaned up during `make clean`.